liteshort.py

```diff
@@ -333,6 +343,8 @@ def main_post():
             and not current_app.config["selflinks"]
         ):
             return response(request, None, "You cannot link to this site")
+        if linking_to_blacklist(request.form["long"]):
+            return response(request, None, "You cannot link to this blacklisted site")
```

Comment:
The `selflinks` option can probably be reworked into the blacklist option. Could you remove lines 341-345 plus the check_self_link method, and just put that check in the linking_to_blacklist function?